src/differential_value_iteration/algorithms/dvi_test.py

```diff
@@ -0,0 +1,45 @@
+"""Tests for basic functioning of DVI algorithms."""
+from absl.testing import absltest
+
+import numpy as np
+
+from differential_value_iteration.algorithms import dvi
+from differential_value_iteration.environments import micro
+
+
+class DVITest(absltest.TestCase):
+
+  def test_dvi_sync_converges(self):
+    environment = micro.mrp1
+    algorithm = dvi.Evaluation(
+        mrp=environment,
+        step_size=.5,
+        beta=.5,
+        initial_r_bar=.5,
+        initial_values=np.zeros(environment.num_states, dtype=np.float32),
+        synchronized=True)
+
+    for _ in range(50):
+      changes = algorithm.update()
+    self.assertAlmostEqual(np.sum(np.abs(changes)), 0.)
```

Comment:
Couple of observations:
- All the *_test.py files are checking for convergence only after the 50th update — which is fine I guess. 
- We're checking the sum of changes here. Perhaps we can change it to mean() like in evaluation_convergence.py to account for number of states?

Reply:
We certainly can. I didn't bother changing this because the MRP for the test is fixed and the sum is more stringent than mean.

This is meant as a basic sanity check that will tell us if we horribly broke something. I think we may want to add some tests that are more targeted.